docs/config.json

```diff
@@ -535,9 +535,17 @@
 	  "entries": [
 	    {
               "title": "Kubernetes Operator",
-              "slug": "/management/dynamic-resources/teleport-operator/",
-              "forScopes": ["oss","enterprise"]
+              "slug": "/management/dynamic-resources/teleport-operator/"
             },
+        {
+          "title": "Kubernetes Operator for Helm-deployed clusters",
+          "slug": "/management/dynamic-resources/teleport-operator-helm/",
+          "forScopes": ["oss","enterprise"]
+        },
+        {
+          "title": "Kubernetes Operator for remote Teleport clusters",
```

Comment:
Maybe it's me but I find "Helm-deployed clusters" vs "remote Teleport clusters" a bit confusing. Can we call these what they literally are, for example:

- Kubernetes Operator in teleport-cluster Helm chart
- Standalone Kubernetes Operator